courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/feat.cols_tf.data.ipynb

```diff
@@ -19,7 +19,9 @@
     "\n",
```

Comment:
<p>Since the header is "Create an Input Line using tf.Data", this should TODO 2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1089/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Thanks @gstripling ,
Updated the TODOs numbering as per objective sequence. Please review.